src/main/java/org/apache/solr/benchmarks/beans/SolrBenchQueryResponse.java

```diff
@@ -0,0 +1,66 @@
+package org.apache.solr.benchmarks.beans;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.lang.invoke.MethodHandles;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicReference;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.solr.client.solrj.request.QueryRequest;
+import org.apache.solr.common.util.NamedList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+// nocommit: javadocs
+public class SolrBenchQueryResponse {
```

Comment:
Nice to have this class to avoid double reading the stream that triggers errors 💪🏼